x/ccv/provider/keeper/keeper.go

```diff
@@ -1158,6 +1162,89 @@ func (k Keeper) DeleteMinimumPowerInTopN(
 	store.Delete(types.MinimumPowerInTopNKey(chainID))
 }
 
+// SetMinStake sets the minimum stake required for a validator to validate
+// a given consumer chain.
+func (k Keeper) SetMinStake(
+	ctx sdk.Context,
+	chainID string,
+	minStake uint64,
+) {
+	store := ctx.KVStore(k.storeKey)
+
+	buf := make([]byte, 8)
+	binary.BigEndian.PutUint64(buf, minStake)
+
+	store.Set(types.MinStakeKey(chainID), buf)
+}
+
+// GetMinStake returns the minimum stake required for a validator to validate
+// a given consumer chain.
+func (k Keeper) GetMinStake(
+	ctx sdk.Context,
+	chainID string,
+) (uint64, bool) {
+	store := ctx.KVStore(k.storeKey)
+	buf := store.Get(types.MinStakeKey(chainID))
+	if buf == nil {
+		return 0, false
+	}
+	return binary.BigEndian.Uint64(buf), true
+}
+
+// DeleteMinStake removes the minimum stake required for a validator to validate
+// a given consumer chain.
+func (k Keeper) DeleteMinStake(
+	ctx sdk.Context,
+	chainID string,
+) {
+	store := ctx.KVStore(k.storeKey)
+	store.Delete(types.MinStakeKey(chainID))
+}
+
+// SetAllowInactiveValidators sets whether inactive validators are allowed to validate
+// a given consumer chain.
+func (k Keeper) SetAllowInactiveValidators(
+	ctx sdk.Context,
+	chainID string,
+	allowed bool,
+) {
+	if allowed {
+		k.AllowInactiveValidators(ctx, chainID)
+	} else {
+		k.DeleteAllowInactiveValidators(ctx, chainID)
+	}
+}
+
+// AllowInactiveValidators sets the flag to signal that inactive validators are allowed to validate
+// a given consumer chain.
+func (k Keeper) AllowInactiveValidators(
```

Comment:
Name too similar with `AllowsInactiveValidators` that is a getter. Might make sense to try other names, e.g., `EnableInactiveValidators` for the setting and `AreInactiveValidatorsEnabled`.

Reply:
Also the name of `SetAllowInactiveValidators` above is slightly peculiar because you can call it with `false` to disallow inactive validators but the method is called `SetAllow`.

Reply:
Renamed both, and also delete